src/box/rt_access.h

```diff
@@ -0,0 +1,53 @@
+#pragma once
+
+#include <stdint.h>
+#include <stdbool.h>
+
+#if defined(__cplusplus)
+extern "C" {
+#endif /* defined(__cplusplus) */
+
+/**
+ * Initialize the user rt access.
+ */
+void
+rt_access_init(void);
+
+/**
+ * Destroy the user rt access.
+ */
+void
+rt_access_free(void);
```

Comment:
I think we still need to call free during deinitialization since we're using `xmalloc`. It's possible to remove `init` and handle initialization when the first privilege is granted, but if we remove `free`, we must ensure that `reset` is called. The question is, where would be the best place to do this?

Reply:
I think it's okay to call `reset` at exit (where you call `free` now).

Reply:
Done.